src/cdfio.F90

```diff
@@ -1251,7 +1254,12 @@ FUNCTION  getvar (cdfile,cdvar,klev,kpi,kpj,kimin,kjmin, ktime, ldiom)
     IF (PRESENT(kimin) ) THEN
        imin=kimin
 
-       ipiglo=getdim(cdfile, cn_x, ldexact=.true.)
+       ipiglo=getdim(cdfile, cn_x, ldexact=.true., kstatus=istatus)
+       ! Else try mesh dimension name
+       IF ( istatus /= 0 ) THEN
+          ipiglo=getdim(cdfile, 'x', ldexact=.true., kstatus=istatus)
+       ENDIF
+
```

Comment:
Why this change is needed ? Was it because of the x_grid_T in the files some times ago ?